frontend/src/app/offers/offer-detail/offer-detail.component.css

```diff
@@ -0,0 +1,35 @@
+.jumbotron{
```

Comment:
I'm no expert in bootstrap. If those styles are not used in v.4 and we're sure we want to migrate to v.4 then IMO it's better to stick to new styles.
Anyway, these styles could be applied everywhere in the project, they're not OfferDetailComponent specific. Please move them to a stylesheet located in a place where it can be common for the whole project.

Reply:
I'm not either, but I believe jumbotron class belongs to v.4 and that's why I've changed it.
https://getbootstrap.com/docs/4.0/components/jumbotron/
I'll move those styles as you wish, but here as I've said before, with jumbotron I wasn't sure how to place offer.image in the background of it and I just checked that again and it doesn't work this way so I'll try to figure out something else. Maybe I should simply remove that jumbotron and just create container for header? I believe using slider is meaningless if we have only one photo for offer.